src/initializers/swagger.initializer.ts

```diff
@@ -5,20 +5,20 @@ import { ArraySizeOptions } from '../options';
 
 export interface SwaggerOptions {
     description?: string;
-    enum?: Record<string, unknown>,
+    example?: unknown;
```

Comment:
There was previously a `string-valued` `example` here, but it should be `unknown/any` and wasn't part of the parent interfaces (see below).

There was also some mixup with commas vs semi-colons. I'm not sure how this passed the compiler/linter.